src/main/java/frc/robot/commands/DriveCommand.java

```diff
@@ -35,7 +35,7 @@ public void initialize() {
   @Override
   public void execute() {
     m_subsystem.drive(m_conditioner.conditionMove(m_controller.getMoveRequest()),
-        m_conditioner.conditionTurn(m_controller.getTurnRequest()));
+        m_conditioner.conditionTurn(-m_controller.getTurnRequest()));
```

Comment:
Well, turning was inverted.  I don't know why, but it was.  I'd guess that some tweaking to Conditioners over the summer did that.

Reply:
It's much more important to understand why than to just patch until working.  Doing that often leads to a big mess.
For instance, it is normal to invert the raw Y axis because of the long history of video games on legacy computers where Y is positive down on the screen.
It is not normal to invert X, so I'd look into this a lot more closely.

Reply:
Jordan fixed this by adding a FunctionalConditioner to the ComposedConditioner that is passed to DriveCommand in RobotContainer.  It reverses turnRequest while leaving moveRequest untouched.